pkgs/by-name/le/legendsviewer-next/frontend.nix

```
@@ -0,0 +1,33 @@
+{
+  version,
+  src,
+  patches,
+
+  nodejs_22,
+
+  buildNpmPackage,
+  nix-update-script,
+}:
+buildNpmPackage rec {
+  pname = "legends-viewer-frontend";
+
+  inherit version src patches;
+
+  nodejs = nodejs_22;
+  npmDepsHash = "sha256-pnhjGlss8U18fK4VKCPiM9kKhmUaJMMBcDOqA/Bexj4=";
+
+  postPatch = ''
+    cd "./LegendsViewer.Frontend/${pname}"
```

```suggestion
    cd ./LegendsViewer.Frontend/legends-viewer-frontend
```
using pname like this is an anti pattern